include/dlr_tvm.h

```diff
@@ -0,0 +1,45 @@
+#ifndef DLR_TVM_H_
+#define DLR_TVM_H_
+
+#include <graph/graph_runtime.h>
+#include "dlr_common.h"
+
+namespace dlr {
+
+/*! /brief Get the paths of the TVM model files.
+ */
+ModelPath get_tvm_paths(const std::string &tar_path);
```

Comment:
Let's use the CamelCase to keep consistency.

ModelPath GetTVMPaths(const std::string& tar_path);

Reply:
fixed